qa/workunits/suse/rbd-snapshot-diff.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -ex
+
+POOL_NAME="rbd"  # using one from create_all_pools_at_once
+IMAGE_NAME="image-$$"
+IMAGE_SIZE="1024"  # MB
+SNAP_NAME="rbd_snap"
+TARGET_DIR="snap-test"
+
+function setup() {
+    rbd create ${POOL_NAME}/${IMAGE_NAME} --size=${IMAGE_SIZE}
+    rbd snap create ${POOL_NAME}/${IMAGE_NAME}@${SNAP_NAME}
+}
+
+function test() {
+    # No diff in the beginning
+    # rbd diff has exit code 0 in both cases, so check is against the command output
+    [[ -z "$(rbd diff ${POOL_NAME}/${IMAGE_NAME})" ]] && echo "No diff as expected"
+    rbd snap ls ${POOL_NAME}/${IMAGE_NAME} | grep ${SNAP_NAME}
+    rbd info ${POOL_NAME}/${IMAGE_NAME}
+    # By default we have: features: layering, exclusive-lock, object-map, fast-diff, deep-flatten
+    # so object-map fast-diff deep-flatten need to be disabled, otherwise rbd map fails
+    rbd feature disable ${POOL_NAME}/${IMAGE_NAME} object-map fast-diff deep-flatten
```

Comment:
Would it make sense to move this line (`rbd feature disable`) into `setup` and then re-enable the features in `teardown`?